### PR TITLE
Dev mariateo burger finished 

### PR DESCRIPTION
**Pull request by MariaTeo**

### CODE REVIEW

**Review by psyhedeliq — APPROVED** [review]
🚀🚀🚀

File: src/_data/burger.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  
+}
```
Comment (RalucaRbc) [inline src/_data/burger.json:3]: Don't think you need this burger.json anymore 😄

**Review by robertoalx — APPROVED** [review]

**Review by RalucaRbc — APPROVED** [review]
Nice, well done Maria 👍

**Review by andreimaff — APPROVED** [review]

**Review by Constantin-Matei — APPROVED** [review]
burger.json?

**Review by vasilemaftita — APPROVED** [review]
Nice work 👍

File: src/_component/burger/burger.pug [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    li.menu__item
+      a.menu__link(href="#open-source")="Open source"
+    li.menu__item
+      a.menu__link(href="#portfolio")="Portfolio"
+    li.menu__item
+      a.menu__link(href="#recommendations")="Recommendations"
+    li.menu__item
+      a.menu__link(href="#reviews")="Reviews"
+    li.menu__item
+      a.menu__link(href="#proud")="I'm proud"
```
Comment (marianzburlea) [inline src/_component/burger/burger.pug]: refactor with an `each` loop

File: src/_component/burger/burger.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  position: fixed;
+  top: 78px;
+  left: 127px;
+  width: 200px;
+}
+
+#burger {
+  display: none;
+
+  &:checked+ul.menu__list {
```
Comment (marianzburlea) [inline src/_component/burger/burger.scss]: add spaced before and after + to increase readability ` + `

File: src/_component/burger/burger.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    display: block;
+  }
+
+  &:checked+ul.menu__list+label {
```
Comment (marianzburlea) [inline src/_component/burger/burger.scss]: same as above

**Review by marianzburlea — CHANGES_REQUESTED** [review]

**Review by biggicode — APPROVED** [review]

**Review by BogdanNegura — APPROVED** [review]

**Pull request closed without merging**